clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -117,17 +121,33 @@ interface FossIdRestService {
         }
 
         /**
-         * Create a FossID service instance for communicating with a server running at the given [url],
+         * Create the [FossIdServiceWithVersion] to interact with the FossID instance running at the given [url],
          * optionally using a pre-built OkHttp [client].
          */
-        fun create(url: String, client: OkHttpClient? = null): FossIdRestService {
+        fun createService(url: String, client: OkHttpClient? = null): FossIdServiceWithVersion {
+            log.info { "The FossID server URL is $url." }
+
             val retrofit = Retrofit.Builder()
-                .apply { if (client != null) client(client) }
+                .apply {
+                    client(
+                        client ?: OkHttpClientHelper.buildClient {
+                            readTimeout(Duration.ofSeconds(60))
+                        }
+                    )
+                }
```

Comment:
Ok, now see the problem why you're using `OkHttpClientHelper.buildClient`: When used from ORT, we'd like to get all the custom settings done in `OkHttpClientHelper`, but we also want to get the adjusted timeout. I'd propose to solve the issue like this:

            val minReadTimeout = Duration.ofSeconds(60)
            val retrofit = Retrofit.Builder()
                .apply {
                    client(
                        client?.run {
                            takeUnless { readTimeoutMillis < minReadTimeout.toMillis() }
                                ?: newBuilder().readTimeout(minReadTimeout).build()
                        } ?: OkHttpClient.Builder().readTimeout(minReadTimeout).build()
                    )
                }
                .baseUrl(url)
                .addConverterFactory(JacksonConverterFactory.create(JSON_MAPPER))
                .build()